src/seedu/addressbook/AddressBook.java

```diff
@@ -508,7 +508,7 @@ private static String executeDeletePerson(String commandArgs) {
             return MESSAGE_INVALID_PERSON_DISPLAYED_INDEX;
         }
         final String[] targetInModel = getPersonByLastVisibleIndex(targetVisibleIndex);
-        return deletePersonFromAddressBook(targetInModel) ? getMessageForSuccessfulDelete(targetInModel) // success
+        return hasPersonDeletedFromAddressBook(targetInModel) ? getMessageForSuccessfulDelete(targetInModel) // success
```

Comment:
This may not be an appropriate change. The original method perform an operation (i.e. deletion) and the return result indicates whether the deletion is successful. This naming change suggests that the method performs a check (i.e. does this person exists?), which is not the case.